.gitignore

```diff
@@ -16,4 +16,8 @@ gitlab/
 
 # ignore vscode files (debug config etc...)
 /.vscode
-/.idea
\ No newline at end of file
+/.idea
+
+# helm chart dependencies
+**/charts/*.tgz
+**/charts/**/requirements.lock
```

Comment:
when we build our helm chart it will download the `oam-kubernetes-runtime` chart tarball and create a `requirements.lock` that has its digest. At first I considered committing this as I think we should treat this dependency as we do go modules, etc., but unfortunately the date time in the `requirements.lock` gets updated every time we `help dependency update` as part of the build. Not committing the `requirements.lock` would only be an issue in the case that a PR ran integration tests pinned to a version of `oam-kubernetes-runtime` then after merge the contents of the `oam-kubernetes-runtime` chart at that version changed (this should never happen) and the Crossplane helm chart was built with the new contents. This is an edge case that we should not encounter.